core/src/main/java/fr/sncf/osrd/api/InfraManager.java

```diff
@@ -152,7 +152,7 @@ private SignalingInfra downloadInfra(
             DiagnosticRecorder diagnosticRecorder
     ) throws InfraLoadException {
         // create a request
-        var endpointUrl = String.format("%sinfra/%s/railjson/", baseUrl, infraId);
+        var endpointUrl = String.format("%sinfra/%s/railjson/?exclude_extensions=false", baseUrl, infraId);
```

Comment:
Almost, the idea is to exclude them (by default they're included)
```suggestion
        var endpointUrl = String.format("%sinfra/%s/railjson/?exclude_extensions=true", baseUrl, infraId);
```